File: src/main/java/net/hectus/neobb/turn/default_game/block/TCauldron.java
package net.hectus.neobb.turn.default_game.block;

import net.hectus.neobb.player.NeoPlayer;
import net.hectus.neobb.turn.default_game.attributes.clazz.NeutralClazz;
import net.hectus.neobb.turn.default_game.attributes.function.AttackFunction;
import org.bukkit.block.Block;

public class TCauldron extends BlockTurn implements AttackFunction, NeutralClazz {
    public TCauldron(NeoPlayer player) { super(player); }
    public TCauldron(Block data, NeoPlayer player) { super(data, player); }

    @Override
    public int cost() {
        return 2;
    }
}

File: src/main/kotlin/net/hectus/neobb/turn/default_game/block/TCauldron.kt
package net.hectus.neobb.turn.default_game.block

import com.marcpg.libpg.storing.Cord
import net.hectus.neobb.player.NeoPlayer
import net.hectus.neobb.turn.default_game.attribute.clazz.NeutralClazz
import net.hectus.neobb.turn.default_game.attribute.function.AttackFunction
import org.bukkit.block.Block

class TCauldron(data: Block?, cord: Cord?, player: NeoPlayer?) : BlockTurn(data, cord, player), AttackFunction, NeutralClazz {
    override val cost: Int = 2
}